examples/pulse_control/2_Native_gate_calibrations.ipynb

```diff
@@ -0,0 +1,418 @@
+    "\n",
+    "Native gate calibrations are available with Rigetti's Aspen M-3.\n",
+    "\n",
+    "First, we will start by initializing a cost tracker and import some libraries."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Use Braket SDK Cost Tracking to estimate the cost to run this example\n",
+    "from braket.tracking import Tracker\n",
+    "t = Tracker().start()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib inline\n",
+    "\n",
+    "from braket.aws import AwsDevice\n",
+    "from braket.native_gates import NativeGateCalibration\n",
+    "from braket.parametric import FreeParameter\n",
+    "from braket.circuits import Circuit, QubitSet\n",
+    "from braket.circuits.gates import Rx, XY\n",
+    "from braket.circuits.serialization import IRType\n",
+    "\n",
+    "import math\n",
+    "from utils.draw_pulse_sequence import draw\n",
+    "import matplotlib.pyplot as plt"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Calibrations are associated with a device so we will need to instantiate an AWS device. By default, the device properties are not automatically populated to limit the amount of data downloaded."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "True\n"
+     ]
+    }
+   ],
+   "source": [
+    "device = AwsDevice(\"arn:aws:braket:us-west-1::device/qpu/rigetti/Aspen-M-3\")\n",
+    "\n",
+    "# Test if the calibrations have been downloaded\n",
+    "print(device._native_gate_calibration is None)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Calibrations are accessible via the `native_gate_calibration` field of the device object. When accessed for the first time, the latest calibrations for all the gates will be downloaded. This takes between XX and YY seconds depending on the performance of your internet connection. The calibrations will be then cached as a dictionnary that is indexed by a tuple formed from a gate object and the qubit number."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "We have downloaded 925 calibrations.\n"
+     ]
+    }
+   ],
+   "source": [
+    "calibrations = device.native_gate_calibration\n",
+    "print(f\"We have downloaded {len(calibrations)} calibrations.\")"
```

Comment:
I think the wording here suggests there are multiple calibration files downloaded. Would something such as, ```"print(f\"The number of configurations in the calibration file is {len(calibrations)} .\")"```